model.py

```diff
@@ -97,7 +101,7 @@ class GPTConfig:
     n_layer: int = 12
     n_head: int = 12
     n_embd: int = 768
-    dropout: float = 0.1
+    dropout: float = 0.0
```

Comment:
Leave a comment saying that this will not be needed after this PR:
https://github.com/pytorch/pytorch/pull/92917 lands